ssl/quic/quic_impl.c

```diff
@@ -3001,9 +3010,11 @@ int ossl_quic_set_write_buffer_size(SSL *ssl, size_t size)
     if (!expect_quic_with_stream_lock(ssl, /*remote_init=*/-1, &ctx))
```

Comment:
Another case where this calls `QUIC_RAISE_NON_NORMAL_ERROR` but below we call ERR_raise. In this case this function is only ever called by the test framework though.

Reply:
We now use QUIC_RAISE_NON_IO_ERROR in expect_quic_with_stream_lock() and below.